### DISCUSSION TITLE
Show statistics in Posts overview table in WP Admin

### DISCUSSION THREAD

**Discussion by dannyvankooten** [ibericode/koko-analytics#204] [Ideas]
For what time period though? Perhaps a setting in the screen options? Or all-time?

**Comment by indysigner** [comment]
Hi @dannyvankooten, as suggested in issue #152 this would be a great feature. If there’s only on default period setting possible, make it 30 days. Thank you! 🙏

**Comment by dannyvankooten** [comment]
Related #258

**Comment by dannyvankooten** [comment]
![image](https://github.com/user-attachments/assets/ba083dbc-0881-4608-a5e8-c70360312453)

![image](https://github.com/user-attachments/assets/6fec94ad-482f-482f-9b2a-ceeb96d823e1)

This has been added to [Koko Analytics Pro](https://www.kokoanalytics.com/pricing/).
